src/content/docs/infrastructure/prometheus-integrations/install-configure-prometheus-agent/troubleshooting-guide.mdx

```diff
@@ -26,6 +33,17 @@ You need, at least, one `job` that is discovering the target in Kubernetes that
 
 If you're using the default configuration on Kubernetes, check that your pod or service has the `prometheus.io/scrape=true` annotation.
 
+By default the Prometheus Agent scrapes metrics only from [Prometheus integrations](/docs/infrastructure/prometheus-integrations/integrations-list/integrations-list-intro). Unless you selected to scrape all the Prometheus endpoints in the cluster, the Prometheus Agent will filter the endpoints to be scraped by using the following Kubernetes labels.
+
+* app.kubernetes.io/name
+* app.newrelic.com/name
```

Comment:
Do we really want to support this? I think we should stick to the standards/recommended labels: https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/

Reply:
Yes, it is just an alternative in case a customer is using app.kubernetes.Io/name for other purposes.
We already agree on it

Also app.newrelic.com/name is the label we will use for scraping additional service if a customers scrapes only metrics from the default Prometheus integrations.